rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -530,13 +527,17 @@ void trophy_manager_dialog::ResizeTrophyIcons()
 	const int dpr = devicePixelRatio();
 	const int new_height = m_icon_height * dpr;
 
+	QList<int> indices;
```

Comment:
Ditto

Reply:
i is used as well as compared as an int in the following code. so size_t is not making any sense here